### PR TITLE
create dark theme friendly variants of the icons

### PR DESCRIPTION
**Pull request by Sivecano**
I wanted dark theme friendly variants of the icons so I added them along with a -d flag for the helper scripts to use them instead of the default images.

not sure, whether this is the way to go. open for constructive criticism.

### CODE REVIEW

**Comment by misterdanb** [comment]
Looks good to me! There is a dark theme mechanism built into gtk, but I think for this kind of tool, the way you did it is the way to go.

Only one thing, could you add the new flag to the help messages of `volumectl` and `lightctl`?

**Comment by Sivecano** [comment]
I did, and I also included the dark variants in the list-resources option of the client

**Comment by misterdanb** [comment]
Great, thank you!

File: volumectl [hunk 1]
```diff
@@ -82,6 +84,7 @@ while getopts ':abD:pg:muh' OPT; do
 		g) opts="$opts --gamma=$OPTARG";;
 		m) dev_type='source';;
 		u) unmute_opt='--unmute';;
+                d) dark=1;;
```
Comment (jirutka) [inline volumectl:87]: Wrong indentation.

File: lightctl [hunk 2]
```diff
@@ -47,11 +48,13 @@ is_float() {
...
 dev=
 optind=1
-while getopts ':e:D:h' OPT; do
+dark=0
+while getopts ':e:D:h:d' OPT; do
 	case "$OPT" in
 		e) exp=$OPTARG;;
 		D) dev=$OPTARG;;
 		h) help; exit 0;;
+                d) dark=1;;
```
Comment (jirutka) [inline lightctl:57]: Wrong indentation.

File: volumectl (hunk 1 above)
Comment (misterdanb) [reply volumectl:87]: Argh, missed that. Gonna fix it later…

**Pull request merged**